spec/lib/workflow/stores/mongo-spec.js

```diff
@@ -44,9 +45,14 @@ describe('Task Graph mongo store interface', function () {
         Constants = helper.injector.get('Constants');
         Errors = helper.injector.get('Errors');
         uuid = helper.injector.get('uuid');
+    };
+    
+    before(function() {
+        resetInjections();
     });
 
     afterEach(function() {
+        resetInjections();
```

Comment:
I don't understand why you need reset injection after each test case, it is unnecessary for from my understanding.